packages/connector-tencent-email/src/types.ts

```diff
@@ -0,0 +1,62 @@
+import { z } from 'zod';
+
+export const FromInfo = z.object({
+  fromAddress: z.string().optional(),
+  fromName: z.string().optional(),
+  replyAddress: z.string().optional(),
+});
+
+export const ValueNames = [
+  'code',
+  'toAddress',
+  'fromAddress',
+  'fromName',
+  'replayAddress',
+  'subject',
+] as const;
+
+export const MailTemplateParameters = z.object({
```

Comment:
Use `camelCase` for zod guard object.
```suggestion
export const mailTemplateParameters = z.object({
```